src/collections/blog/2022/2022-06-13-aaditya-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,54 @@
+subtitle: "Nothing but a magical experience"
+date: 2022-06-13 08:00:00 -0630
+author: Aaditya Narayan Subedy
+thumbnail: ./aaditya-subedy-layer5-intern.png
+category: Programs
+description: "Aaditya Narayan Subedy's journey from a Contributor at Layer5 to getting an GSoC internship at Layer5"
+tags:
+ - Community
+ - Projects
+ - News
+type: Blog
+published: true
+---
+import { Link } from "gatsby";
+import { BlogWrapper } from "../../Blog.style.js";
+import aadityaFirstPr from "./aaditya-first-pr.png"
+import aadityaPresenting from "./aaditya-presenting.png"
+
+
+<BlogWrapper>
+
+### Early Interest in Coding 
+I grew up with an immense interest in computers. I used to be fascinated by how programs run on the computer that makes such an impact on the people using it. This always motivated me to work in the field of computers. 
+
+As I grew up I was introduced to a broad spectrum of coding languages that enabled us to make the software that we used on the computer. These languages piqued my interest even more, and I started to explore and make projects. The more I understood them the more interesting it got. Slowly I realized that I just don’t want to make projects that satisfy my needs. I want to build something that will be on the internet and will be used by people around the world. With this goal in my mind, I started exploring and hit a gemstone that is Open source. I would be working on real-world programming solutions that will be used by people all around the world. This helped me to get started and got me excited about contributing to open-source projects. 
+
+### First PR at Layer5 
+I still remember the first issue I worked on for Layer5 was the auto labeler on a pull request. I saw a good first issue tag and still not knowing what the task was, I stated my interest to work on the issue. The issue required knowledge of GitHub Actions which at that time I didn’t even know there was such a thing. After getting the issue assigned, I started researching what GitHub Actions were. I found out that these actions helped to automate workflows that help in building, testing, and deploying software. After getting the basics down I looked for if there are any GitHub Actions already present for the auto labeling of the PR, and to my surprise, there were a couple of them. 
+
+So before actually making a PR, I experimented with the Actions extensively in my private repo. I remember making about 20-25 PR in my private repo just to get the auto labeler working fine. After I verified I got the desired results, I made a PR. I also got to discuss my approach in the development call. I was so nervous at that time since that was the first meeting I attended where I had to make people understand what I did and what approaches I took. I was quite thrilled when my code got merged. I was enthralled to see that my code was working and being useful to the community. This motivated me to make more and more contributions. 
```

Comment:
```suggestion
So before actually making a PR, I experimented with the actions extensively in my private repo. I remember making about 20-25 PR in my private repo to get the auto labeler working fine. After verifying that I got the desired results, I made a PR. I also got to discuss my approach in the development call. I was so nervous at that time since that was the first meeting I attended where I had to make people understand what I did and the strategies I used. I was quite thrilled when my code got merged. I was enthralled to see that my code was working and being useful to the community. This motivated me to make more and more contributions. 
```

Reply:
Link to Meshery Development Call to https://layer5.io/community/calendar